lib/kitchen/patches/string.rb

```diff
@@ -17,6 +17,6 @@ def uncapitalize
   # @return [String]
   #
   def kebab_case
-    strip.downcase.gsub(/[^(\w\s)]/, '').gsub(/\s/, '-')
+    strip.downcase.gsub(/®/, ' r').gsub(/[^(\w\s)]/, '').gsub(/\s/, '-')
```

Comment:
We don't know if there are other symbols that should be supported, so I made change just for the trademark symbol. If any other appear, I will create more general solution.